### ISSUE TITLE
`COLORFGBG` was set incorrectly by `setDarkBackground()`

### ISSUE THREAD

**Issue by eatradish** [LingmoOS/lingmo-terminal#8]
In Lingmo OS's default theme, the `echo $COLORFGBG` is returned as `15;0` (this will give the program reporting the value of this variable, which now with the terminal background color as black)

```cpp
    // this is not strictly accurate use of the COLORFGBG variable.  This does not
    // tell the terminal exactly which colors are being used, but instead approximates
    // the color scheme as "black on white" or "white on black" depending on whether
    // the background color is deemed dark or not
    QString backgroundColorHint = _hasDarkBackground ? QLatin1String("COLORFGBG=15;0") : QLatin1String("COLORFGBG=0;15");
```
The ` _hasDarkBackground` variable seems to be modified only by the `setDarkBackground` function, and I can only see this call in `Session *TermWidgetImpl::createSession(QWidget* parent)` with it set to true.

```cpp
Session *TermWidgetImpl::createSession(QWidget* parent)
{
// balabala...
session->setDarkBackground(true);
// balabala...
}
```

In other words, if I understand you correctly, the program will always consider the background color of the terminal to be black.

**Comment by elysia-best** [comment]
Fixed in https://github.com/LingmoOS/lingmo-terminal/pull/10
